spec/integration/outputs/index_version_spec.rb

```diff
@@ -0,0 +1,100 @@
+require_relative "../../../spec/es_spec_helper"
+require "logstash/outputs/elasticsearch"
+
+
+describe "Versioned indexing", :integration => true, :version_greater_than_equal_to_2x => true do
+  require "logstash/outputs/elasticsearch"
+
+  before :each do
```

Comment:
Can you change this section to be:

```ruby
let(:es) { get_client }

before(:each) do
  # Delete all templates first.
  # Clean ES of data before we start.
  es.indices.delete_template(:name => "*")
  # This can fail if there are no indexes, ignore failure.
  es.indices.delete(:index => "*") rescue nil
  es.indices.refresh
end 
```

you will also have to generally replace `@es` with `es`